alfresco-transform-imagemagick/alfresco-transform-imagemagick/src/main/java/org/alfresco/transformer/executors/ImageMagickCommandExecutor.java

```diff
@@ -35,9 +37,18 @@
  */
 public class ImageMagickCommandExecutor extends AbstractCommandExecutor
 {
-    private static final String ROOT = "/usr/lib64/ImageMagick-7.0.7";
-    private static final String DYN = ROOT + "/lib";
-    private static final String EXE = "/usr/bin/convert";
+    private static String ROOT;
+    private static String DYN;
+    private static String EXE;
```

Comment:
Could/should these be "final" (rather than "static") ?

Reply:
It can't be final in this case as final vars need to be initialised and then won't be able to reassigned it with the new value at runtime.

Reply:
R u sure - can be initialised once ?

Reply:
It can be initialised to null, but then won't be able to reassigned with the new value at runtime since it's final.

Reply:
Sorry my bad,  just saw "(rather than "static")", if it's no longer static then it can be done.